### PR TITLE
VEN-717 | Winter storage areas page: Detailed row

### PR DESCRIPTION
**Pull request by TatuArvela**
## Description :sparkles:

* Add detailed row to winter storage areas list

### Related changes ⛓️

* Move `HarborDetails` terms into `common.terminology`
* Rename `HarborMapLinks` to `MapLinks`
* Move `HarborCard` `harborPlaceholder` to common `placeholderImage`, use in `HarborDetails`

### Code quality improvements 🧹

* Adjust linting settings to fix all icon related lint warnings
* Fix all remaining lint warnings

### Closes :no_good_woman:

* [VEN-717](https://helsinkisolutionoffice.atlassian.net/browse/VEN-717): FE - Winter storage areas page: Detailed row

### Automated tests :gear:️

* Created snapshot test for WinterStorageAreaDetails
* Updated test data and snapshots for winterStorageAreaList utils test

### Manual testing :construction_worker_man:

* Check that the new component is visible and looks as expected
* Check Storybook stories for WinterStorageAreaList and WinterStorageAreaDetails

### CODE REVIEW

File: src/features/winterStorageAreaList/winterStorageAreaDetails/WinterStorageAreaDetails.tsx [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+}: WinterStorageAreaDetailsProps) => {
+  const { t, i18n } = useTranslation();
+  const address = `${streetAddress}, ${zipCode} ${municipality}`;
+  const imageSrc = imageFile ? imageFile : placeholderImage;
+  const serviceMapUrl = `${process.env.REACT_APP_SERVICE_MAP_URI}${servicemapId}`;
+
+  return (
+    <Grid colsCount={4}>
+      <div className={styles.column}>
+        <img className={styles.image} src={imageSrc} alt="map" />
```
Comment (Nizar-Rahme) [inline src/features/winterStorageAreaList/winterStorageAreaDetails/WinterStorageAreaDetails.tsx:41]: I think it's better to add `alt` value to the translation files.

File: src/features/winterStorageAreaList/winterStorageAreaDetails/WinterStorageAreaDetails.tsx [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+            {t('common.terminology.serviceMap')}
+          </ExternalLink>
+        </Section>
+        <MapLinks maps={maps} />
+      </div>
+      <div className={styles.column}>
+        <Section title={t('common.terminology.maxWidth').toUpperCase()}>
+          <Text>{formatDimension(maxWidth, i18n.language)}</Text>
+        </Section>
+        <Section title={t('common.terminology.mooring').toUpperCase()}>-</Section>
```
Comment (Nizar-Rahme) [inline src/features/winterStorageAreaList/winterStorageAreaDetails/WinterStorageAreaDetails.tsx:58]: We might leave a `TODO` comment here for future generations.

File: src/features/winterStorageAreaList/winterStorageAreaDetails/WinterStorageAreaDetails.tsx [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+export type WinterStorageAreaDetailsProps = {
+  imageFile: WinterStorageAreaData['imageFile'];
+  maps: WinterStorageAreaData['maps'];
+  maxWidth: WinterStorageAreaData['maxWidth'];
+  municipality: WinterStorageAreaData['municipality'];
+  servicemapId: WinterStorageAreaData['servicemapId'];
+  streetAddress: WinterStorageAreaData['streetAddress'];
+  zipCode: WinterStorageAreaData['zipCode'];
+};
```
Comment (Nizar-Rahme) [inline src/features/winterStorageAreaList/winterStorageAreaDetails/WinterStorageAreaDetails.tsx:22]: `Pick` might come in handy here.

File: src/features/winterStorageAreaList/winterStorageAreaDetails/WinterStorageAreaDetails.tsx (hunk 1 above)
Comment (TatuArvela) [reply src/features/winterStorageAreaList/winterStorageAreaDetails/WinterStorageAreaDetails.tsx]: Great idea, I did this for all instances of `alt` in the codebase (3 in total)

File: src/features/winterStorageAreaList/winterStorageAreaDetails/WinterStorageAreaDetails.tsx (hunk 2 above)
Comment (TatuArvela) [reply src/features/winterStorageAreaList/winterStorageAreaDetails/WinterStorageAreaDetails.tsx]: Added, also for HarborDetails

File: src/features/winterStorageAreaList/winterStorageAreaDetails/WinterStorageAreaDetails.tsx (hunk 3 above)
Comment (TatuArvela) [reply src/features/winterStorageAreaList/winterStorageAreaDetails/WinterStorageAreaDetails.tsx]: Changed, also for HarborDetails

**Pull request merged**